### PR TITLE
Api List Order Detail Of A Order

### PR DESCRIPTION
**Pull request by at-vudang95**

### CODE REVIEW

File: routes/api.php [hunk 1]
```diff
@@ -17,6 +17,7 @@
     Route::put('/users/me', 'Api\UserController@update');
     Route::get('/users/me', 'Api\UserController@show');
     Route::post('/orders', 'Api\OrderController@store');
+    Route::get('/orders/{id}/getItems', 'Api\OrderController@show');
```
Comment (ghost) [inline routes/api.php]: I don't understand `getItems`?

Comment (at-vudang95) [reply routes/api.php]: This is get order item of another order

Comment (at-vudang95) [reply routes/api.php]: https://github.com/AT-PHPIntership/food-market-order/pull/214/files

Comment (at-vudang95) [reply routes/api.php]: How do I should? Thank you

Comment (ghost) [reply routes/api.php]: Just use `/orders/{id}/items`

**Comment by ghost** [comment]
LGTM 🌊

**Pull request merged**